frontend/src/metabase/collections/components/BulkActions.styled.tsx

```diff
@@ -3,15 +3,12 @@ import styled from "@emotion/styled";
 import Card from "metabase/components/Card";
 import Button from "metabase/core/components/Button";
 import { alpha, color } from "metabase/lib/colors";
-import { NAV_SIDEBAR_WIDTH } from "metabase/nav/constants";
 import { space } from "metabase/styled-components/theme";
 
-export const BulkActionsToast = styled.div<{ isNavbarOpen: boolean }>`
+export const BulkActionsToast = styled.div`
   position: fixed;
   bottom: 0;
   left: 50%;
-  margin-left: ${props =>
-    props.isNavbarOpen ? `${parseInt(NAV_SIDEBAR_WIDTH) / 2}px` : "0"};
```

Comment:
This PR puts a [CSS container](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_containment/Container_queries) around the ItemsTable, so this adjustment is no longer needed (and actually makes the BulkActionsToast appear too far to the right).